README.md

```diff
@@ -52,7 +52,10 @@ giving us both centralized and local logs.  (Redundancy is nice).
 		flag.Parse()
 
 		if graylogAddr != "" {
-			gelfWriter, err := gelf.NewWriter(graylogAddr)
```

Comment:
I was just thinking the other day, after reading about conventions for [package names](https://golang.org/doc/effective_go.html#package-names), that `NewWriter` was the wrong name anyway. `Dial` for consistency with other stuff sounded good until I did more research, and now I'm realizing that we seem to be going about this all wrong; GELF and its framing is all application-level protocol stuff and should just run over any stream or datagram connection, all of which we could have the client supply to us as something that implements `Write([]byte) (len, error)`. Of course, we'd need some internal configuration to deal with setting appropriate framing based on whether we're told it's a stream or datagram (or "packet", as they say in `net` package world) connection. Things like reconnecting should be handled by a different package whose `Writer` the `go-gelf` package would use.